_posts/2024-04-30-aspnet-efcore.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: distill
+title: Asp.net 8.0 and Next with Docker-compose
+description: How to create a project with Asp Dotnet 8.0 and Next with Docker-compose
+tags: Docker-compose With dotnet and next
+giscus_comments: true
+date: 2024-04-18
+featured: false
+
+authors:
+  - name: Usman Jamil Bhatti
+    url: "https://github.com/usman-jamil"
+    affiliations:
+      name: Allied Consultants
+  - name:  Sheikh Luqman
+    url: "https://github.com/luqmant51"
+    affiliations:
+      name: Allied Consultants
+  - name: Zuhaib Sohail
+    url: "https://github.com/ZuhaibSohail163"
+    affiliations:
+      name: Allied Consultants
+
+_styles: >
+  .fake-img {
+    background: #bbb;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    box-shadow: 0 0px 4px rgba(0, 0, 0, 0.1);
+    margin-bottom: 12px;
+  }
+  .fake-img p {
+    font-family: monospace;
+    color: white;
+    text-align: left;
+    margin: 12px 0;
+    text-align: center;
+    font-size: 16px;
+  }
+
+---
+
+ASP.net 8.0 and Entity Core Framework with postgres sql and docker compose
+
+Create Web Api Project 
```

Comment:
If these 2 are heading then mark them as I think line 42 is title of this blog?

Reply:
```
Not heading just a description
```